src/external/atomsbase.jl

```diff
@@ -107,6 +107,9 @@ function AtomsBase.atomic_system(lattice::AbstractMatrix{<:Number},
             Atom(atomic_symbol(element), position; kwargs...)
         end
     end
+    # Check consistency of the atomic masses.
+    all(getfield.(atomsbase_atoms, :atomic_mass) .== atomic_mass.(atoms)) ||
```

Comment:
This is not where the problem is, actually. What I'm concerned about is `parse_system` above, where an warning like line 56 should be issued. I can take care of this before the merge if you want.

Reply:
Yes, that's exactly what I meant.